lib/sender.js

```diff
@@ -4,15 +4,31 @@
 
 const net = require('net');
 const tls = require('tls');
-const { randomFillSync } = require('crypto');
 
 const PerMessageDeflate = require('./permessage-deflate');
 const { EMPTY_BUFFER } = require('./constants');
 const { isValidStatusCode } = require('./validation');
 const { mask: applyMask, toBuffer } = require('./buffer-util');
+const { randomBytes } = require('crypto');
 
 const mask = Buffer.alloc(4);
 
+const RANDOM_POOL_SIZE = 8192;
+const RANDOM_POOL_REFRESH = 1024;
+
+let randomPool = Buffer.alloc(RANDOM_POOL_SIZE);
```

Comment:
Maybe Buffer.alloc(0) here to allow the check on line 100 to trigger and not waste some heap?